systemtest/src/test/java/io/strimzi/systemtest/security/NetworkPoliciesST.java

```diff
@@ -177,26 +177,28 @@ void testNetworkPoliciesOnListenersWhenOperatorIsInSameNamespaceAsOperands() {
         );
 
         LOGGER.info("Initialize producers and consumers with access to the Kafka using plain and tls listeners");
-        KafkaClients kafkaClientsWithAccessPlain = ClientUtils.getDefaultClientBuilder(testStorage)
+        KafkaClients kafkaClientsWithAccessPlain = ClientUtils.getInstantPlainClientBuilder(testStorage)
+            .withUsername(testStorage.getUsername())
```

Comment:
why do you need to specify username here? From what I see this should be only plain client